src/cockpit/channels/dbus.py

```diff
@@ -330,7 +331,9 @@ async def do_call(self, message):
             # actually, should send the fields from the message body
             self.send_message(error=[error.name, [error.message]], id=cookie)
         except Exception as exc:
-            self.send_message(error=['python.error', [str(exc)]], id=cookie)
+            printable = '\n'.join(traceback.format_exception(exc))
+            logger.warning("do_call(%s): generic exception %s", message, printable)
```

Comment:
Alternatively, use `logger.exception`? https://docs.python.org/3/library/logging.html#logging.Logger.exception

I don't think you need the `printable part them. You do ofcourse for `send_message` so gains seem minimal.

Reply:
But that logs at level "error", which we cannot do right now -- if we do that, we need to fix the pybridge/cockpit.js/storage page to not trigger this error if stratis is not found. See https://github.com/cockpit-project/cockpit/pull/18143#issuecomment-1375638126

Reply:
Wait, ignore me -- that was a different error. I'll look into that, thanks!

Reply:
The journal output looks pretty much identical except for s/WARNING/ERROR/. Indeed the code diff is minimal, but at least a little more idiomatic:
```
--- src/cockpit/channels/dbus.py
+++ src/cockpit/channels/dbus.py
@@ -331,8 +331,8 @@ class DBusChannel(Channel):
             # actually, should send the fields from the message body
             self.send_message(error=[error.name, [error.message]], id=cookie)
         except Exception as exc:
+            logger.exception("do_call(%s): generic exception", message)
             printable = '\n'.join(traceback.format_exception(exc))
-            logger.warning("do_call(%s): generic exception %s", message, printable)
             self.send_message(error=['python.error', [printable]], id=cookie)
 
```

So I'll push this.